### PR TITLE
[WIP] Staff Page

### PR DESCRIPTION
**Pull request by jamierocks**
Currently this page does not display all of the same information as the [docs page](https://docs.spongepowered.org/stable/en-GB/about/staff.html) does. For that reason, this page is intended to supplement the docs page - giving a more graphic thanks to our team <3

---

Screenshot

---

The `staff.json` file has been produced from the structure present in the docs page, I have tried my best to get accurate GitHub profiles - though some may be incorrect, it would be a good idea to check your entries :)

(for future reference) I used the following Python script to get profile images:
```py
from urllib.request import urlopen
from bs4 import BeautifulSoup
import json

staff = []

with open('staff.json') as json_data:
    staff = json.load(json_data)

for group in staff['current']:
    for member in group['members']:
        if not member['img']:
            print(member['name'])
            resp = urlopen(member['url'])
            soup = BeautifulSoup(resp, "html.parser")

            member['img'] = soup.find('meta', {'property': 'og:image'})['content']

with open('staff.json', 'w') as outfile:
    json.dump(staff, outfile, indent=4)
```

### CODE REVIEW

**Comment by phase** [comment]
Can the background for the leaders use



(https://www.spongepowered.org/assets/img/backgrounds/sponsors-content.jpg)

or a variation of it? Some solid colors in the same color scheme would work too.

**Comment by kashike** [comment]
Can we not uppercase people's names?

**Comment by jamierocks** [comment]
@kashike [wrote](https://github.com/SpongePowered/SpongeHome/pull/150#issuecomment-413723174)
> Can we not uppercase people's names?

We can, currently that list was taken directly from the docs page. If you'd like any changes ping me on IRC :)

File: staff.json [hunk 1]
```diff
@@ -0,0 +1,322 @@
...
+                },
+                {
+                    "name": "JBYoshi",
+                    "img": "https://avatars2.githubusercontent.com/u/12983479?s=400&v=4",
+                    "url": "https://github.com/JBYoshi"
+                },
+                {
+                    "name": "Luck",
+                    "img": "https://avatars2.githubusercontent.com/u/296868?s=400&v=4",
+                    "url": "https://github.com/Luck"
```
Comment (lucko) [inline staff.json]: `https://github.com/lucko` :)

**Comment by Spongy** [comment]
A preview for this pull request is available at https://spongy.github.io/SpongeHome-PRs/150/.

Here are some links to the pages that were modified:

- added: https://spongy.github.io/SpongeHome-PRs/150/staff

**Comment by phase** [comment]
Can we make overflow circles center? It looks a little odd on the left.

And what if the retired names were in alphabetical order vertically? I think that would make more sense since the centered text suggests you should read it top to bottom.

File: staff.json [hunk 2]
```diff
@@ -0,0 +1,322 @@
...
+                    "url": "https://github.com/Grinch"
+                },
+                {
+                    "name": "Lemonous",
+                    "img": "https://avatars1.githubusercontent.com/u/2099030?s=400&v=4",
+                    "url": "https://github.com/afriestad"
+                },
+                {
+                    "name": "Vectrix",
+                    "img": "https://avatars2.githubusercontent.com/u/22254644?s=280&v=4",
```
Comment (phase) [inline staff.json]: `https://avatars0.githubusercontent.com/u/12867785?s=460&v=4`
`https://github.com/connorhartley`

File: staff.json (hunk 1 above)
Comment (phase) [reply staff.json]: Avatar: `https://avatars2.githubusercontent.com/u/8352868?s=460&v=4`

**Review by phase — CHANGES_REQUESTED** [review]
Some avatars need updating.

The [Sponsors](https://www.spongepowered.org/sponsors) page also needs updating.

Also the names of some people are quite long and look awkward (check `mosemister` at the bottom), and on smaller resolutions it jumbles up.

2018-08-17-102356_1920x1080_scrot

**Pull request closed without merging**